contents/teams/marketing/objectives.mdx

```diff
@@ -1,24 +1,122 @@
+
+**Contributors:** Joe working with Sales & CS team
+
+**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/186)
+
+**We'll know we're successful when:** We're hitting our SLAs and our CSAT has either improved or remained stable. CSAT is currently 96% satisfaction. 
+
+### Support CS in their sales-led activities (Joe)
+
+**Rationale:** Sales & CS is changing how it works. There's an opportunity to support Sales in getting larger deals, and to improve processes around how incoming deals work. Deals = Growth. 
+
+**Contributors:** Joe working with Sales & CS team
+
+**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/193)
+
+**We'll know we're successful when:** We're effectively processing deals of all sizes and up/cross-selling users into new features. CS team feel they have everything they need to close 90% of current deals. 
+
+### Become default analytics tool for new AI startups (Lior)
+
+**Rationale:** Our strategy is to get in first and AI startups are generally engineering-led, so strongly within our ICP. A significant portion of recent YC batches are AI startups and they have specific needs we need to cater to.
+
+**Contributors:** Lior
+
+**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/190)
+
+**We'll know we're successful when:** We have 50 customers using the Langfuse integration and strong usage among AI startups in upcoming YC batches.
+
+### Finish alternatives guides (Andy)
+
+**Rationale:** We know these improve visibility and drive signups. We have a long list of guides we can ship and a proven formula we can repeat.
+
+**Owner:** Andy
+**Contributors:** Andy, Ian, Lior
+
+**What we'll ship:** [See issue](https://github.com/PostHog/posthog.com/issues/7517#issuecomment-2015715230)
+
+**We'll know we're successful when:** We've shipped priority guides.
+
+### Scale newsletter (James)
+
+**Rationale:** We've successfully reached 10k subs and people like the newsletter. We think it can be a big driver of word of mouth growth if we can scale it further.
+
+**Contributors:** James Temperton, Andy, Ian, Lior, James Hawkins. 
+
+**What we'll ship: (WIP)** [See issue](https://github.com/PostHog/meta/issues/192)
+- Transition to 1 new issue per week
+- Improve content promotion on Twitter and LinkedIn
+- Trade way more recommendations with other newsletters
+- Get at least 1 new (or old) post to go viral
+- Improve visibility on the website
+- Do more issues that use our network
+- Land a big interview
+- Continue newsletter sponsorships
+
+**We'll know we're successful when:** We can consistently ship great newsletters every week while maintaining a strong open rate (45% or more) and subscriber growth is accelerating.
+
+### Level up branding (Lottie)
+
+**Rationale:** People love our brand, but it will go stale if don't refresh and create new things that spark joy and conversation.
```

Comment:
```suggestion
**Rationale:** People love our brand, but it will go stale if don't refresh and create new things that spark joy and conversation.

```